sites/partners/src/components/listings/PaperListingForm/sections/AdditionalFees.tsx

```diff
@@ -69,34 +72,79 @@ const AdditionalFees = (props: AdditionalFeesProps) => {
               )}
             />
           </Grid.Cell>
+        </Grid.Row>
+        <GridRow>
           <Grid.Cell>
-            <Field
+            <FieldGroup
               register={register}
-              type={"currency"}
-              prepend={"$"}
-              {...defaultFieldProps(
-                "depositMin",
-                t("listings.depositMin"),
-                props.requiredFields,
-                errors,
-                clearErrors
-              )}
-            />
-          </Grid.Cell>
-          <Grid.Cell>
-            <Field
-              register={register}
-              type={"currency"}
-              prepend={"$"}
-              {...defaultFieldProps(
-                "depositMax",
-                t("listings.depositMax"),
-                props.requiredFields,
-                errors,
-                clearErrors
-              )}
+              type="radio"
```

Comment:
issue: This new "Deposit Range" vs "Fixed Deposit" option should only appear if the listing is "Non regulated". It should stay as the original layout and not have the "Deposit Type" question if is of type "Regulated"